integration-tests-jvm/google-secret-manager/pom.xml

```diff
@@ -58,6 +58,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct</artifactId>
```

Comment:
You'll need to add the `-deployment` dependency into the `virtualDependencies` profile.

Reply:
yes, I thought I  added it there, (but it might have been a different PR), thanks!

Reply:
fixed